### PR TITLE
Works and About Section

### PR DESCRIPTION
**Pull request by hajnaloltyan**
Portfolio site using Template 1 - Works and About sections for mobile

- Created the HTML structure for the required sections
- Added the styles for them using Grid and Flexbox

- [Live Demo Link](https://hajnaloltyan.github.io/portfolio-mv)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,51 +30,336 @@ <h3 class="header__logo-text">
...
+            <p class="project__description"">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="flex project__languages">
+              <li class="project__language__item">HTML</li>
+              <li class="project__language__item">CSS</li>
+              <li class="project__language__item">JavaScript</li>
+            </ul>
+
```
Comment (Kamba56) [inline index.html:125]: - [ ] Please note that you are expected to stick to design as much as possible. This is including spacings between elements. The spaces between the elements in your cards looks less than it is on the template. KIndly improve on this please.
### Your design
image
### Figma
image

File: style.css [hunk 2]
```diff
@@ -73,23 +92,204 @@ li {
...
+  display: block;
+  width: 102px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 141%;
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+  padding: 12px;
+  margin-bottom: 12px;
```
Comment (Kamba56) [inline style.css:192]: - [ ] Consider adding a hover and pressed state to all your buttons as it is on the figma template. You can use the `:hover` and `:active` pseudoclasses for this.
image

File: index.html [hunk 3]
```diff
@@ -27,51 +30,336 @@ <h3 class="header__logo-text">
...
+        </article>
+      </section>
+
+      <!-- About Section -->
+
+      <section class="about flex">
+        <h2 class="about__title default-padding">
+          About Myself
+        </h2>
+
```
Comment (Kamba56) [inline index.html:277]: - [ ] Kindly reduce the width of this heading so it matches the figma template. You are required to stick to design as much as possible, this includes width, height, fonts, sizes, margins, and paddings.
### FIgma
image
### Your design
image

File: index.html [hunk 4]
```diff
@@ -14,6 +14,9 @@
     <title>Portfolio</title>
```
Comment (Kamba56) [inline index.html:14]: - [ ] I noticed your getting started section is of no help to the user. Kindly populate it so users can easily set up a local copy of your work.
image

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @hajnaloltyan, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed
- [x] Great job deploying your page

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (hajnaloltyan) [reply index.html:277]: @Kamba56 Hi, I'm trying to add a screenshot of Figma, the design there looks the same as mine, the Text is in one line
Screenshot from 2023-03-24 21-52-13

Comment (hajnaloltyan) [reply index.html:277]: @Kamba56 I don't know how to put a picture here, it shows up broken, but my design looks like the Figma design, the Text is in 1 line

**Review by braincee — APPROVED** [review]
Hi @hajnaloltyan  , I am @braincee, your Code Reviewer.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

youre-approved-shirley-auto-group

OPTIONAL CHANGES
---

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @braincee._

---

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 3 above)
Comment (Kamba56) [reply index.html:277]: I'm sorry about that. I made a copy of the old design, I didn't know the design has been changed. I apologize.

**Pull request merged**